packages/terra-clinical-detail-view/src/DetailList.jsx

```diff
@@ -20,25 +20,47 @@ const propTypes = {
     PropTypes.objectOf(DetailListItem),
     PropTypes.arrayOf(PropTypes.objectOf(DetailListItem)),
   ]),
+
+  /**
+   * Indicates if the DetailList should be a description list for label-value pairs.
+   */
+  isDescriptionList: PropTypes.bool,
```

Comment:
Please include info on the `<LabelValueView>` component as an option to use for paired data since that component is already implemented the `<dt>` and `<dd>` tags.